dev-games/aseprite/aseprite-1.3.5.ebuild

```diff
@@ -96,19 +97,6 @@ src_prepare() {
 }
 
 src_configure() {
-	# -Werror=strict-aliasing, -Werror=odr, -Werror=lto-type-mismatch
-	# https://bugs.gentoo.org/924692
-	# https://github.com/aseprite/aseprite/issues/4413
-	#
-	# There is a bundled skia that fails with ODR errors. When excluding just
-	# skia from testing, aseprite itself fails with strict-aliasing, and when
-	# that is disabled, fails again with ODR and lto-type-mismatch issues.
-	#
-	# There are a lot of issues, so don't trust any fixes without thorough
-	# testing.
```

Comment:
Was this tested? I think not. At minimum, you haven't patched the QA violation in the skia archive, nor have you upgraded the skia revision.